### DISCUSSION TITLE
Fail to Compile After Reinstall VS Code

### DISCUSSION THREAD

**Discussion by Alaa-Aim** [vortigont/espem#33] [Q&A] [answered]
@vortigont , Hi , after reinstall VS Code and PIO extension, I had and an error every-time I want to build /compile, as below 

Compiling .pio\build\espem\lib5f0\WiFi\WiFiScan.cpp.o
In file included from .pio/libdeps/espem/EmbUI/EmbUI/EmbUI.h:464,
                 from espem/espem.cpp:10:
.pio/libdeps/espem/EmbUI/EmbUI/ui.h: In member function 'void Interface::div(const String&, const String&, const T&, const String&, const String&, const ArduinoJson6201_F1::JsonObject&)':
.pio/libdeps/espem/EmbUI/EmbUI/ui.h:657:23: error: 'ArduinoJson6200_F1' does not name a type; did you mean 'ArduinoJson6201_F1'?
         nobj.operator ArduinoJson6200_F1::JsonVariant().shallowCopy(params);
                       ^~~~~~~~~~~~~~~~~~
                       ArduinoJson6201_F1
In file included from .pio/libdeps/espem/EmbUI/EmbUI/EmbUI.h:464,
                 from espem/interface.cpp:2:
.pio/libdeps/espem/EmbUI/EmbUI/ui.h: In member function 'void Interface::div(const String&, const String&, const T&, const String&, const String&, const ArduinoJson6201_F1::JsonObject&)':
.pio/libdeps/espem/EmbUI/EmbUI/ui.h:657:23: error: 'ArduinoJson6200_F1' does not name a type; did you mean 'ArduinoJson6201_F1'?
         nobj.operator ArduinoJson6200_F1::JsonVariant().shallowCopy(params);
                       ^~~~~~~~~~~~~~~~~~
                       ArduinoJson6201_F1
In file included from .pio/libdeps/espem/EmbUI/EmbUI/EmbUI.h:464,
                 from espem/main.cpp:12:
.pio/libdeps/espem/EmbUI/EmbUI/ui.h: In member function 'void Interface::div(const String&, const String&, const T&, const String&, const String&, const ArduinoJson6201_F1::JsonObject&)':
.pio/libdeps/espem/EmbUI/EmbUI/ui.h:657:23: error: 'ArduinoJson6200_F1' does not name a type; did you mean 'ArduinoJson6201_F1'?
         nobj.operator ArduinoJson6200_F1::JsonVariant().shallowCopy(params);
                       ^~~~~~~~~~~~~~~~~~
                       ArduinoJson6201_F1
Compiling .pio\build\espem\lib5f0\WiFi\WiFiServer.cpp.o
Compiling .pio\build\espem\lib5f0\WiFi\WiFiUdp.cpp.o
*** [.pio\build\espem\src\main.cpp.o] Error 1
*** [.pio\build\espem\src\interface.cpp.o] Error 1
*** [.pio\build\espem\src\espem.cpp.o] Error 1
==================================================================== [FAILED] Took 13.76 seconds ====================================================================

Environment    Status    Duration
-------------  --------  ------------
espem          FAILED    00:00:13.764

Any advise ? thanks in advanced  

Edit : I managed to build the code after changing "ArduinoJson6200_F1" in  both EmbUI/EmbUI/ui.h & EmbUI/EmbUI/ui.cpp  to "ArduinoJson6201_F1"

**Comment by vortigont** [comment, marked as answer]
Hi @Alaa-Aim,
fixed in [8acee07](https://github.com/vortigont/EmbUI/commit/8acee073ea723ab9b39c8874f84b486d2642a9ce)

**Reply by Alaa-Aim** [reply]
Many Thanks @vortigont
